draft-add-requirements.md

```diff
@@ -232,23 +277,22 @@ this threat model may have limited value.
 This section lists requirements that flow from the above sections.
 
 | Requirement | Description |
-| R1.1 | Discovery MUST provide a local network the ability to announce to clients a set of, or absence of, equivalent resolvers. |
-| R1.2 | Discovery MUST provide a resolver the ability to announce to clients a set of, or absence of, equivalent resolvers. |
+| R1.1 | Discovery SHOULD provide a local network the ability to announce to clients a set of, or absence of, designated resolvers. |
+| R1.2 | Discovery SHOULD provide a resolver the ability to announce to clients a set of, or absence of, designated resolvers. |
 | R1.3 | Discovery MUST support at least one encrypted DNS protocol. |
 | R1.4 | Discovery SHOULD support all standardised encrypted DNS protocols. |
-| R2.1 | Networks MUST be able to announce one or more equivalent encrypted DNS resolvers using existing mechanisms such as DHCPv4, DHCPv6, IPv6 Router Advertisement, and the Point-to-Point Protocol. |
-| R2.2 | The format for resolver information MUST be specified such that provisioning mechanisms defined outside of the IETF can advertise encrypted DNS resolvers. |
+| R2.1 | Networks SHOULD be able to announce one or more designated encrypted DNS resolvers using existing mechanisms such as DHCPv4, DHCPv6, IPv6 Router Advertisement, and the Point-to-Point Protocol. |
+| R2.2 | The format for resolver designation SHOULD be specified such that provisioning mechanisms defined outside of the IETF can advertise encrypted DNS resolvers. |
+| R2.3 | This format SHOULD convey, at minimum, the information the client needs to make contact with each designated resolver. |
+| R2.4 | This format MAY convey more information. |
 | R3.1 | When discovery is instantiated from a resolver (R1.2), that resolver MAY be encrypted or not. |
```

Comment:
Discovery is initiated by the client but not by the resolver. What do you mean by "resolver may be encrypted or not" ?
Are you trying to imply "The communication with the resolver to discover a designated resolver MAY be encrypted or not" ?

Reply:
It's supposed to say that in resolver-identified designation, the resolver that is doing the designating might be accessed over an unencrypted channel (Do53) or encrypted (DoH etc). I will amend.

Reply:
R3.1 now says "In resolver-identified designation (R1.2), the communication with the designator MAY be encrypted or not, depending on the capability of the resolver."

Reply:
works for me.